.babelrc.js

```diff
@@ -14,7 +14,6 @@ module.exports = {
   }]],
   "plugins": [
     "./resources/inline-invariant",
-    "syntax-async-functions",
```

Comment:
Not required anymore and was moved to archive:
https://github.com/babel/babel-archive/tree/master/packages/babel-plugin-syntax-async-functions